languages/ruby/exercises/concept/floating-point-numbers/.meta/example.rb

```diff
@@ -1,19 +1,20 @@
 module SavingsAccount
-
-  NEGATIVE_RATE = -3.213
-  SMALL_POSITIVE_RATE = 0.5
-  MEDIUM_POSITIVE_RATE = 1.621
-  LARGE_POSITIVE_RATE = 2.475
+  RATE = {
+    NEGATIVE: -3.213,
+    SMALL_POSITIVE: 0.5,
+    MEDIUM_POSITIVE: 1.621,
+    LARGE_POSITIVE: 2.475
+  }
 
   def self.interest_rate(balance)
     if balance.negative?
-      NEGATIVE_RATE
+      RATE[:NEGATIVE]
     elsif balance < 1000
-      SMALL_POSITIVE_RATE
+      RATE[:SMALL_POSITIVE]
     elsif balance < 5000
-      MEDIUM_POSITIVE_RATE
+      RATE[:MEDIUM_POSITIVE]
     else
-      LARGE_POSITIVE_RATE
+      RATE[:LARGE_POSITIVE]
```

Comment:
This needs reverting. We shouldn't introduce hashes in this example.

Reply:
Sorry about that, I misread you message from two days ago about introducing hashes. I somehow  missed the `not` part

Reply:
That's ok :)

Reply:
If you revert that, I'll merge it. Then you can follow up on @ErikSchierboom's extra point later. I want to shorten the PR review times. Once we hit "good enough", let's get things onto master and create issues for extra bits that need adding.

Reply:
Done, sorry about the delay :(